### ISSUE TITLE
curl: (60) SSL certificate problem: unable to get local issuer certificate

### ISSUE THREAD

**Issue by abithakt** [hashbang/hashbang.sh#130]
Hi, 

When I try to run the command `curl https://hashbang.sh | gpg -o hashbang.sh` I get the following message:

```
gpg: WARNING: no command supplied.  Trying to guess what you mean ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
gpg: processing message failed: Unknown system error
```

**Comment by paulwratt** [comment]
confirmed with wget:
```
--2019-07-07 23:49:13--  https://hashbang.sh/server/stats
Resolving hashbang.sh (hashbang.sh)... 104.131.13.197, 2604:a880:800:10::1909:a001
Connecting to hashbang.sh (hashbang.sh)|104.131.13.197|:443... connected.
ERROR: The certificate of ‘hashbang.sh’ is not trusted.
ERROR: The certificate of ‘hashbang.sh’ doesn't have a known issuer.
```
NOTE: no such problems getting a result via web browser.

**Comment by daurnimator** [comment]
I sshed into the server and fixed it. There was a real mess of things going on. I moved all the stuff that seemed useless to a directory `~/old-thought-useless-in-2019-07/`.

I set up a new `hashbangsh.service.d-override.conf` systemd override file that sets up the correct certificate location, which is the `letsencrypt` folder in the home directory. I believe `./cert.sh` can be used to renew the cert in that subfolder.
